engines/account/config/routes.rb

```diff
@@ -3,9 +3,9 @@
     namespace :v1 do
       post '/signups', to: 'signups#create'
       get '/signups/:token', to: 'signups#show'
+      post '/signups/:token', to: 'signups#complete'
```

Comment:
I find weird to `POST` to the same URL: `/signups/`. 
What is the reason to not use `/signups/complete`?